pkg/consensus/consensus.go

```diff
@@ -319,7 +317,7 @@ func (s *service) broadcast(p payload.ConsensusPayload) {
 	switch p.Type() {
 	case payload.PrepareRequestType:
 		pr := p.GetPrepareRequest().(*prepareRequest)
-		pr.minerTx = *s.txx.Get(pr.transactionHashes[0]).(*transaction.Transaction)
+		pr.tx = *s.txx.Get(pr.transactionHashes[0]).(*transaction.Transaction)
```

Comment:
It makes no sense, you're picking some random transaction. Previously miner tx was broadcasted this way because it was special, it couldn't be put into the mempool like regular one.